.gitignore

```
@@ -2,3 +2,5 @@ utils/node_modules
 
 # IDE files
 .DS_Store
+settings.json
+.env
```

Adding a star pattern here will also take care of files like `.env.development` 
```suggestion
.env*
```